ui/client/components/toolbars/status/buttons/CustomActionButton.tsx

```diff
@@ -25,8 +25,8 @@ export default function CustomActionButton(props: Props) {
       ? <img alt={`custom-action-${action.name}`} src={action.icon} />
       : <DefaultIcon/>
 
-  const statusName = processStatus.name
-  const isDisabled = !action.allowedProcessStates.map(s => s.name).includes(statusName)
+  const statusName = processStatus?.name
+  const isDisabled = processStatus === null ? true : !action.allowedStateStatusNames.includes(statusName)
```

Comment:
is it needed? I mean, action.allowedStateStatusNames.includes(null) is false? or not? :)

Reply:
:+1:   ur right, it's unnecessary, it's false anyways